context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for anyone who wants to create images using written instructions.
```

Comment:
I would note that this is not just any kind of written instructions. Compare Context Free to some kind of Turtle graphics for example.